dev-tools/cmd/asset/asset.go

```diff
@@ -79,7 +80,7 @@ func main() {
 		}
 	}
 
-	encData, err := asset.EncodeData(string(data))
+	encData, err := asset.EncodeData(strings.Replace(string(data), "\r", "", -1))
```

Comment:
Do you want to add a note here on why it's here to make sure if someone touches the code in the future he knows?

Reply:
Good idea :slightly_smiling_face: